Redis/tests/System/V1/CloudRedisClientTest.php

```diff
@@ -31,111 +34,81 @@
  */
 class CloudRedisClientTest extends TestCase
 {
-    protected static $grpcClient;
-    protected static $projectId;
-    private static $hasSetUp = false;
+    const LOCATION_ID = 'us-central1';
+    const TIER = Tier::BASIC;
+    const MEMORY_SIZE_GB = 1;
 
-    public function clientProvider()
-    {
-        self::setUpBeforeClass();
-
-        return [
-            [self::$grpcClient]
-        ];
-    }
+    private static CloudRedisClient $client;
+    private static string $parent;
+    private static string $instanceId;
+    private static string $instanceName;
 
-    public static function setUpBeforeClass(): void
+    /**
+     * @beforeClass
+     */
+    public static function setUpTestFixtures()
     {
-        if (self::$hasSetUp) {
-            return;
-        }
-
         $keyFilePath = getenv('GOOGLE_CLOUD_PHP_TESTS_KEY_PATH');
         $keyFileData = json_decode(file_get_contents($keyFilePath), true);
+        $projectId = $keyFileData['project_id'];
```

Comment:
`json_decode` can return `false`, should we check for it?

Reply:
Good call, done!